Documentation/devicetree/bindings/media/bcm2835-unicam.txt

```diff
@@ -46,9 +46,9 @@ Example:
 		reg = <0x7e801000 0x800>,
 		      <0x7e802004 0x4>;
 		interrupts = <2 7>;
-		clocks = <&clocks BCM2835_CLOCK_CAM1>;
-		clock-names = "lp";
-
+		clocks = <&clocks BCM2835_CLOCK_CAM0>,
```

Comment:
`It's a minor point, but this example has CSI1 using `CLOCK_CAM0`, whereas the real .dtsi files have CSI0 using `CLOCK_CAM0` and CSI1 using `CLOCK_CAM1`.

Reply:
Upstream has switched to yaml, so this file is an orphan anyway.

Reply:
I'll fix it up any way.  Update coming shortly.